src/main/java/org/reaktivity/nukleus/kafka/internal/stream/KeyMessageDispatcher.java

```diff
@@ -27,7 +27,7 @@
 
 public class KeyMessageDispatcher implements MessageDispatcher
 {
-    private final UnsafeBuffer buffer = new UnsafeBuffer(new byte[0]);
+    final UnsafeBuffer buffer = new UnsafeBuffer(new byte[0]);
```

Comment:
`protected final`

Reply:
Generally I prefer to take the most restrictive scope possible so that's why I left it package private. I agree protected has the advantage of better indicating intended usage (by subclasses), so happy to change it to that.